### PR TITLE
Fix: assets_version instead of asset_version

### PR DESCRIPTION
**Pull request by sebastianblum**
The name of the function is assets_version, see
https://github.com/symfony/symfony/blob/2.7/src/Symfony/Bundle/TwigBundle/Extension/AssetsExtension.php

### CODE REVIEW

**Comment by javiereguiluz** [comment]
@sebastianblum the reason for this issue is that the assets management has change a lot with the introduction of the new Asset component. We have the documentation for the component ready but it has to be merged: https://github.com/symfony/symfony-docs/pull/4987

Once it's merged, we'll review all the doc to update the all the configuration options and code samples.

**Comment by stof** [comment]
@javiereguiluz this change is actually right. ``asset_version`` does not exist in Symfony 2.6 (it is the new function introduced for the Asset component). In Symfony 2.6 and older, the function is ``assets_version`` (it still exist in 2.7 but it is deprecated)

File: reference/twig_reference.rst [hunk 1]
```diff
@@ -113,12 +113,12 @@ Returns a public path to ``path``, which takes into account the base path set
 for the package and the URL path. More information in
 :ref:`book-templating-assets`. For asset versioning, see :ref:`ref-framework-assets-version`.
 
-asset_version
+assets_version
 ~~~~~~~~~~~~~
```
Comment (xabbuh) [inline reference/twig_reference.rst]: You have to add one ~ here to match the new headline length.

**Comment by WouterJ** [comment]
@sebastianblum can you please add the missing tilde (`~`)? After that, this PR can be merged.

**Comment by sebastianblum** [comment]
@WouterJ sorry for the missing ~, now it should be fine.
Should I rebase the commit?

**Comment by WouterJ** [comment]
No, everything is perfect now. (I'm just waiting for GitHub to solve its ddos attack so I can merge...)

**Comment by WouterJ** [comment]
I finally managed to get through, thanks Sebastian!

**Pull request closed without merging**